index.js

```diff
@@ -1,8 +1,12 @@
 /** DON'T EDIT THIS FILE WHICH WAS CREATED BY 'scripts/generate-index.js'. */
```

Comment:
Should revert the changes in this file?

I dont see the `scripts/generate-index.js` file around though, so shall we remove this comment?